README.md

```diff
@@ -80,18 +80,18 @@ After following either one of these steps the site will be available at `http://
 ### Deployment
 
 The project is built automatically by GitHub Actions whenever the `master` branch receives a new commit. The `master` branch
-itself should not be deployed, as it only contains the source files. The build version of the website is available as the
+itself should not be deployed, as it only contains the source files. The built version of the website is available on the
```

Comment:
I think "as" is okay, btw, as the branch is the direct representation of the built version 🙃